condor_watch_q.py

```diff
@@ -451,8 +457,12 @@ def table_by_event_log(self, abbreviate_path_components=False):
         for d in dont_include:
             for row in rows:
                 row.pop(d)
-
+        for r in rows:
+            for x in r:
+                 if r.get(x) == 0:
+                      r[x] = "-"
```

Comment:
I'm actually not convinced that `-` is better than `0` for empty entries, but I'll take this up with @bbockelm and TJ.

Reply:
:)

For now, it's about consistency with `condor_q`.  If you can get the default changed in `condor_q` to show `0`, I'd be happy to keep as-is here.

To some extent, we are prioritizing consistency over improving the outputs.  Improving the outputs of `condor_q`is going to be a bit of a longer slog...